### DISCUSSION TITLE
Multiple "Accounts"

### DISCUSSION THREAD

**Discussion by thilina15** [aldinokemal/go-whatsapp-web-multidevice#104] [Q&A]
can i connect multiple accounts to this server?

**Comment by aldinokemal** [comment]
Hey @thilina15 , This app isn't set up to work with multiple accounts, but you can copy it to a different place and run it on a different port. Good luck!

**Reply by berkbirkan** [reply]
In this alternative scenario you presented, it becomes almost impossible to automate this structure. Isn't there a way to log in to multiple WhatsApp accounts in a single container?

**Reply by eduanmoldeep** [reply]
There is only one solution that does allows that -https://waha.devlike.pro/

**Comment by aldinokemal** [comment]
Multiple account already supported
